sources/documents/RTB/Verbali/Interni/VerbaleInterno_2024-04-20/local.tex

```diff
@@ -0,0 +1,26 @@
+%%%%%%%%%%%%%%
+%  COSTANTI  %
+%%%%%%%%%%%%%%
+
+% In questa prima parte vanno definite le 'costanti' utilizzate soltanto da questo documento.
+% Devono iniziare con una lettera maiuscola per distinguersi dalle funzioni.
+
+\newcommand{\DocTitle}{Verbale Riunione 2024-04-20}
+\newcommand{\DocVersion}{0.0.1}
+\newcommand{\DocApprovazione}{TODO}
+\newcommand{\DocUso}{Interno}
+\newcommand{\DocDistribuzione}{
+	\Committente{} \\
+	Gruppo \GroupName{}
+}
+
+% La descrizione del documento
+\newcommand{\DocDescription}{
+	La riunione è stata organizzata con l'obiettivo di analizzare e valutare l'andamento dello \glossario{sprint} appena concluso. Inoltre, il team ha pianificato la prossima iterazione ed esaminato gli \glossario{Issue Tracking System} individuati come possibili alternative all'\glossario{ITS} di \glossario{GitHub}.
```

Comment:
```suggestion
	La riunione è stata organizzata con l'obiettivo di analizzare e valutare l'andamento dello \glossario{sprint} appena concluso. Inoltre, il team ha pianificato la prossima iterazione ed esaminato gli \glossario{Issue Tracking System} (a seguire \glossario{ITS}) individuati come possibili alternative all'\glossario{ITS} di \glossario{GitHub}.
```

Reply:
Questa aggiunta secondo te è strettamente necessaria?

Reply:
Perché nel Glossario su ITS ci sarà già un link a Issue Tracking System

Reply:
necessaria no, ma semplifica la lettura senza il passaggio per il glossario
nel caso lo gestirei comuqnue solo alla prima occorrenza del documento anche in caso di lettura non romanzata

Reply:
quindi la prima volta che appare Issue Tracking System tu vorresti segnare tra parentesi ITS?

Reply:
si una nota come quella sopra del tipo che da lì in poi la stessa cosa viene identificata come ITS

Reply:
Ok ci sta, modifico la prima occorrenza quindi